docs/conf.py

```diff
@@ -30,7 +30,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 #extensions = ['sphinx.ext.autodoc']
 
-extensions = ['djangocms', 'sphinx.ext.intersphinx', 'sphinx.ext.todo']
+extensions = ['djangocms', 'sphinx.ext.intersphinx']
```

Comment:
please don't do this, sphinx.ext.todo is actually useful
this is the correct fix for the sphinx issue https://github.com/divio/django-cms/commit/7979a0620248bc3d82dd35414a342cf7062e001b and it's already in develop